builtin/providers/aws/resource_aws_route53_zone_association_test.go

```diff
@@ -0,0 +1,218 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/service/route53"
+)
+
+func TestAccRoute53ZoneAssociation(t *testing.T) {
```

Comment:
Could you please add a `_basic` suffix here? We're starting to adopt that scheme for tests, otherwise we can't run `TestAccRoute53ZoneAssociation` in isolation without also running `TestAccRoute53ZoneAssociationWithRegion`. `TestAccRoute53ZoneAssociationWithRegion` should then be `TestAccRoute53ZoneAssociation_WithRegion`.